Models/User.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace blogResearchNetCore.Models
+{
+    public class User : BaseEntity
+    {
+        public string Username {get; set; }
+        public string Password { get; set; }
+        public UserStatus IsStatus { get; set; }
```

Comment:
Datatype of this field is UserStatus, not Bool. So that I do not think prefixing it with "Is"